src/tests/baseservices/exceptions/unhandled/unhandled.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace TestUnhandledException
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            if (args.Length != 0)
+            {
+                throw new Exception("Test");
+            }
+
+            List<string> lines = new List<string>();
+
+            Process testProcess = new Process();
+
+            testProcess.StartInfo.FileName = Environment.ProcessPath;
+            testProcess.StartInfo.Arguments = Environment.CommandLine + " throw";
+            testProcess.StartInfo.UseShellExecute = false;
+            testProcess.StartInfo.RedirectStandardError = true;
+            testProcess.ErrorDataReceived += (sender, line) => 
+            {
+                Console.WriteLine($"\"{line.Data}\"");
+                if (!string.IsNullOrEmpty(line.Data))
+                {
+                    lines.Add(line.Data);
+                }
+            };
+
+            testProcess.Start();
+            testProcess.BeginErrorReadLine();
+            testProcess.WaitForExit();
+            testProcess.CancelErrorRead();
+
+            int expectedExitCode;
+            if ((Environment.OSVersion.Platform == PlatformID.Unix) || (Environment.OSVersion.Platform == PlatformID.MacOSX))
+            {
+                expectedExitCode = 128 + 6;
+            }
+            else if (TestLibrary.Utilities.IsNativeAot)
+            {
+                expectedExitCode = unchecked((int)0xC0000409);
+            }
+            else
+            {
+                expectedExitCode = unchecked((int)0xE0434352);
+            }
+
+            if (expectedExitCode != testProcess.ExitCode)
+            {
+                Console.WriteLine($"Wrong exit code 0x{testProcess.ExitCode:X8}");
+                return 101;
+            }
+
+            if (Regex.Match(lines[0], @"Unhandled exception[.:] System\.Exception\: Test", RegexOptions.IgnoreCase) == Match.Empty)
```

Comment:
Feel free to change the native AOT string to `Unhandled exception` and use a simple exact match here instead.